batch/batch/client.py

```diff
@@ -91,19 +92,27 @@ def wait(self):
             status = self.status()
             if status['jobs']['Created'] == 0:
                 return status
-            j = random.randrange(2 ** i)
+            j = random.randrange(math.floor(1.1 ** i))
             time.sleep(0.100 * j)
-            # max 5.12s
-            if i < 9:
+            # max 4.45s
+            if i < 64:
                 i = i + 1
 
 
 class BatchClient:
-    def __init__(self, url=None, api=api.DEFAULT_API):
+    def __init__(self, url=None, timeout=None, token_file=None, headers=None):
         if not url:
             url = 'http://batch.default'
         self.url = url
-        self.api = api
+        if token_file is None:
+            token_file = os.path.expanduser('~/.hail/batch/token')
```

Comment:
The authentication is Hail services wide, so I think just `~/.hail/token`.

Reply:
done